cognite/neat/_rules/transformers/_converters.py

```diff
@@ -233,95 +234,167 @@ class ToExtension(RulesTransformer[DMSRules, DMSRules]):
     def __init__(
         self,
         new_model_id: DataModelIdentifier,
-        org_name: str | None = None,
-        mode: Literal["extension", "solution"] = "extension",
+        org_name: str = "My",
+        type_: Literal["enterprise", "solution"] = "enterprise",
+        mode: Literal["read", "write"] = "read",
+        dummy_property: str = "GUID",
     ):
         self.new_model_id = DataModelId.load(new_model_id)
         if not self.new_model_id.version:
-            raise ValueError("Version is required for the new model.")
+            raise NeatValueError("Version is required for the new model.")
```

Comment:
My personal preference is to only do validation in the transform method, not in the `__init__`.